factory/random.py

```diff
@@ -22,3 +24,6 @@ def reseed_random(seed):
     """Reseed factory.fuzzy's random generator."""
     r = random.Random(seed)
     set_random_state(r.getstate())
```

Comment:
`r.getstate()` should be stored in a variable since it is reused below. Beside that it does make sense. LGTM